config/nginx/docker.conf

```diff
@@ -0,0 +1,21 @@
+server {
```

Comment:
Github won't let me comment on it but I'm confused why nginx.conf was moved to config/nginx/vagrant.conf